### PR TITLE
Dylan october 10

### PR DESCRIPTION
**Pull request by Dhubbs**

### CODE REVIEW

**Comment by Dhubbs** [comment]
integrates the functionality of https://github.com/SocialTensor/SuperEnsembleClassifier

File: constants.py [hunk 1]
```diff
@@ -6,6 +6,14 @@
 
 DB_NAME = "image_generation_service"
 
+MODEL_CLASSIFICATION_URL = "https://superensembleclassifier-production.up.railway.app/classify"
```
Comment (toilaluan) [inline constants.py]: should we have this variable as an environment variable

File: services/image_generation_service.py [hunk 2]
```diff
@@ -348,11 +348,26 @@ def check_validator(hotkey):
     async def get_validators(self) -> List:
         return list(self.available_validators.keys())
 
+    async def get_model_classification(self, model_name: str, prompt: str) -> str:
+        if model_name == "SuperEnsemble":
+            try:
+                response = requests.post(MODEL_CLASSIFICATION_URL, json={"prompt": prompt})
```
Comment (toilaluan) [inline services/image_generation_service.py]: we should add a small value for timeout here

File: constants.py (hunk 1 above)
Comment (toilaluan) [reply constants.py]: if it's not defined in env variables, just ignore this step and return random model name, wdyt?

**Comment by Dhubbs** [comment]
Please add to env
CLASSIFIER_URL="https://superensembleclassifier-production.up.railway.app/classify"

**Pull request merged**